server/manifestmergetool/src/test/java/com/defold/manifestmergetool/ManifestMergeToolTest.java

```diff
@@ -115,7 +115,7 @@ protected void createDefaultFiles() throws IOException {
             this.main = new File(contentRoot, "builtins/manifests/ios/Info.plist");
             this.libraries = new ArrayList<File>();
             this.libraries.add(new File(contentRoot, "builtins/manifests/ios/InfoLib.plist"));
-        } else if (platform == Platform.HTML5) {
+        } else if (platform == Platform.WEB) {
```

Comment:
Fixes in this file are not related to the issue, but I wasn't able to finish the tests without fixing this.